config/permissions.tmpl.yml

```diff
@@ -35,6 +35,7 @@
 - ems-type:gce_network
 - ems-type:hawkular
 - ems-type:hawkular_datawarehouse
+- ems-type:monitoring
```

Comment:
@moolitayer if we do (and we're not planning to) why wouldn't it be a different manager with an hawkular name?
It would be an OpenShift provider with an Hawkular alert manager instead of a Prometheus manager.